components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/workflow/ApplicationRegistrationWSWorkflowExecutor.java

```diff
@@ -170,7 +142,76 @@ public void complete(WorkflowDTO workFlowDTO) throws WorkflowException {
 		return null;
 	}
 
-    public String getServiceEndpoint() {

```

Comment:
This method is not properly formatted according to the WSO2 coding style. Please format accordingly.